### ISSUE TITLE
scancode/licensecheck: (new) scancode/licensecheck_raw

### ISSUE THREAD

**Issue by RobertBerger** [priv-kweihmann/meta-sca#3590]
You added a new `scancode_raw`, which is nice, but it looks like it contains only the info of those scans, where `sca` found a problem. Can you please add all of them? 

In order to review a couple of things, it would be really helpful to have all of the license scans available.

Same for licensecheck.

**Comment by RobertBerger** [comment]
Let me rephrase this.

I am not sure based on what criteria it outputs things ;)

**Comment by RobertBerger** [comment]
... but glibc is missing

**Comment by priv-kweihmann** [comment]
did the tool run for glibc?
Basically the export is done for every tool, where scancode module is active (to be checked by `bitbake glibc -e | grep ^SCA_ACTIVE_MODULES`).
When it runs it does perform the initial check on **all** source files (the raw report) the classification and linting itself is done after packaging and on a package basis, which simply means that the files packaged in the pkg or that could be tracked back to source files are taken from the raw report and further processed.
The export is done no matter if there have been findings or not - so the only thing that could go wrong is that the task isn't run at all on the recipe

**Comment by priv-kweihmann** [comment]
@RobertBerger I found the reason for glibc not being taken into account -> https://github.com/priv-kweihmann/meta-sca/blob/d1c57c015bd0ccd713d671c84feb81a61c3ca6c8/classes/sca-blacklist.bbclass#L104.
To avoid dependency loops everything in `BASE_DEFAULT_DEPS` will be filtered out automatically.

So for now this is expected behavior - I will work on a better filtering somewhat soon

**Comment by RobertBerger** [comment]
OK - so I am not seeing crazy shit ;) Good!
